electron/install.js

```diff
@@ -264,9 +264,9 @@ function removeInstallationLogFile() {
 
 async function setupDarwin(ipcChannel) {
   removeInstallationLogFile();
-  console.log(appendInstallationLog('Creating required directories'));
-  await createDirectory(`${paths.dotOperateDirectory}`);
-  await createDirectory(`${paths.dotOperateDirectory}/temp`);
+  // console.log(appendInstallationLog('Creating required directories'));
+  // await createDirectory(`${paths.dotOperateDirectory}`);
+  // await createDirectory(`${paths.dotOperateDirectory}/temp`);
```

Comment:
This _may_ cause an issue if /.operate/temp is required?

Reply:
ln 139, install.js:
```
async function installTendermintUnix() {
  const cwd = process.cwd();
  process.chdir(paths.tempDir);
```

this function tries to change to `paths.tempDir` which is `.operate/temp` but the folder may not exist, so we should probably try to make it in the `setupDarwin` function

something like
```
if(!fs.existsSync(paths.tempDir)){
  fs.mkdir(paths.tempDir);
}
```

Reply:
Simplest option is to uncomment the line, as we have "createDirectory" method already